.github/workflows/sycl_update_gpu_driver.yml

```diff
@@ -0,0 +1,58 @@
+name: Update GPU driver
+
+on:
+  schedule:
+    - cron:  '0 3 * * 2'
+  workflow_dispatch:
+
+jobs:
+  update_driver_linux:
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update dependencies file
+      run: |
+        version="$(python3 devops/scripts/update_drivers.py linux)"
+        echo 'NEW_DRIVER_VERSION='$version >> $GITHUB_ENV
+    - name: Create Pull Request
+      env:
+        BRANCH: ci/update_gpu_driver-linux-${{ env.NEW_DRIVER_VERSION }}
+        LLVMBOT_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        cd $GITHUB_WORKSPACE
+        # Set fake identity to fulfil git requirements
+        git config --global user.name "iclsrc"
+        git config --global user.email "ia.compiler.tools.git@intel.com"
+        git checkout -B $BRANCH
+        git add -u
+        git commit -m "[GHA] Uplift GPU RT version for Linux CI" || exit 0   # exit if commit is empty
```

Comment:
I added an exit to the commits to simply bail if the commit is empty. Thus avoiding pushing empty branches or opening empty PRs.

Reply:
I believe if no changes you won't be producing anything empty. git commit will exit with "nothing to commit, working tree clean" and exit code 1.

Reply:
@pvchupin  - I tested both with and without the `|| exit 0` and while the commit fails in both cases, if we don't have the `exit 0` at the top of the .yml then it moves on and the empty branch is pushed, which we definitely don't want.

I could change the error code so instead of `exit 0` it is `exit 1`.  Then the action will be marked as FAILED in the GitHub UI. My thinking was since there was nothing to do we are simply done - that isn't a failure situation.

Reply:
I see. That's fine.